tests/baselines/reference/skipJSDocParsing/see-ParseForTypeInfo-file.ts.diff

```diff
@@ -28,7 +28,6 @@
 -                                "pos": 9,
```

Comment:
This seems unintended?

Reply:
This is a misleading diff; the file is already a diff, so it's a diff of a diff when viewed on github. The only prop going away is `modifierFlagsCache` below.